ironfish/src/sdk.ts

```diff
@@ -37,6 +37,7 @@ export class IronfishSdk {
   logger: Logger
   metrics: MetricsMonitor
   internal: InternalStore
+  hosts: HostsStore
```

Comment:
Do the node + sdk still need to construct the HostsStore? It looks like it's constructed at a lower level in the AddressManager now.